css/style.css

```diff
@@ -0,0 +1,1248 @@
+  }
+
+  .section-arrows-white:before,
+  .section-arrows-orange:before,
+  .section-arrows-black:before {
+    position: absolute;
+    content: "";
+    display: block;
+    width: 20%;
+    height: 43px;
+    top: 0;
+    left: 40%;
+  }
+
+  .section-arrows-orange:before {
+    background: url("/img/triangle-orange.svg") no-repeat;
+  }
+
+  .section-arrows-black:before {
+    background: url("/img/triangle-black.svg") no-repeat;
+  }
+
+  .section-arrows-white:before {
+    background: url("/img/triangle-white.svg") no-repeat;
+  }
+}
+
+/*
+MENU
+*/
+
+.menu {
+  position: fixed;
+  width: 100%;
+  padding-bottom: 0.5em;
+  border-bottom: 2px solid #d4d4d4;
+  background-color: rgba(255, 255, 255, 0.95);
+  z-index: 1000;
+}
+
+.menu a {
+  display: inline-block;
+  width: 30%;
+  color: #7c8387;
+  font-weight: bold;
+  padding-top: 0.5em;
+}
+
+@media screen and (min-width: 50em) {
+  .menu {
+    position: static;
+    padding: 0.8em calc(20% - 3em);
+  }
+
+  .menu a {
+    padding-top: 0.1em;
+    width: 16%;
+  }
+}
+
+/*
+HEADER & LOGO
+*/
+
+.header-content {
+  padding: 3em 0;
+  background-size: cover;
+  background-image: url(/img/header-background.jpg);
+}
+
+.logo {
+  opacity: 1;
+  height: 4em;
+  width: 4em;
+  margin: 1em auto 1em;
+  padding-top: 1em;
+  padding-bottom: 1em;
+  background: #fd6632;
+}
+
+.header-logo-text strong {
+  font-weight: normal;
+}
+
+.curve-underline {
+  padding-bottom: .1em;
+  background-size: cover;
+  background: url("/img/orange-underline.svg") no-repeat;
+  background-position: bottom;
+}
+
+@media screen and (min-width: 50em) {
+  h1 {
+    font-size: 2.666em;
+  }
+}
+
+
+/*
+SERVICES COMPONENTS
+*/
+
+.services-container {
+  padding: 2.5em .5em 1em;
+  text-align: center;
+  background-color: #fd6632;
+}
+
+.services-text,
+.contact-text {
+  display: inline-block;
+  padding: .5em 0;
+  line-height: 1.4;
+}
+
+.services-category {
+  padding: 1.333em;
+}
+
+.services-header {
+    min-width: 225px;
+  padding: .5em 0;
+}
+
+[data-title="ux-design"]:after {
+  content: "\e60b";
+}
+
+[data-title="laboratory"]:after {
+  content: "\e600";
+}
+
+[data-title="development"]:after {
+  content: "\e601";
+}
+
+.services-header:after {
+  display: block;
+  width: 2em;
+  height: 2em;
+  margin: .4em auto 0;
+  padding-top: .5em;
+  padding-bottom: .5em;
+  font-family: 'ico-fonts';
+  font-size: 2em;
+  text-align: center;
+  color: #2e2e2e;
+  background-color: #fff;
+  border-radius: 50%;
+}
+
+@media screen and (min-width: 50em) {
+  .services-category-container {
+    display: -webkit-flex;
+    display: flex;
+    -webkit-flex-wrap: wrap;
+    flex-wrap: wrap;
+    -webkit-justify-content: space-around;
+    justify-content: space-around;
+    -webkit-align-content: center;
+    align-content: center;
+  }
+
+  .services-text,
+  .contact-text {
+    padding: 1em;
+  }
+
+  .services-category {
+    width: 33%;
+  }
+}
+
+
+
+/*
+PRICING COMPONENTS
+*/
+
+[class^="pricing"] {
```

Comment:
Note that this will _only_ get elements where the class string starts with `pricing`. If you had an element that was something had a class string of `foo bar pricing-baz`, these rules would not be applied.

Additionally, all of the `text-align: center`s in this set of rules are unnecessary because they all inherit from `.services-container` which is set to center the text.